kuma/settings/common.py

```diff
@@ -882,6 +882,7 @@ def pipeline_one_scss(slug, **kwargs):
             "js/libs/prism/prism-wasm.js",
             "js/libs/prism/prism-line-highlight.js",
             "js/libs/prism/prism-line-numbers.js",
+            "js/components/mathml.js",
```

Comment:
A recent conversation with Peter made me have thoughts about this:
Our current Kuma way of writing JS (including what's already in the codebase, my PRs and this PR) is brittle, i.e. it's barely tested, non-type checked and has implicit dependencies (also called `querySelector`s). Now any synchronous code that gets added to the main bundle has the potential of erroring and thereby preventing any other script to run. The cherry on top is that this behavior is release-mode exclusive as these scripts listed here get separate `<script>`-tags in dev.

So given all that my recommendation for this PR is to have mathml be a separate script. The drawback is that it takes longer for MathML to be displayed. But given that it is also dynamically loading in CSS, I think we haven't chosen the fast-path for this anyway. So imo we can take the hit.